setup.py

```diff
@@ -78,10 +78,11 @@
         ]
     },
     install_requires=[
-        # This can not be installed from pip, on Fedora you need to install
-        # python{2,3}-rpm package containing the module.
+        # This can not be installed from pip on system Python, on Fedora
+        # you need to install python{2,3}-rpm package containing the module.
         # See: https://github.com/rpm-software-management/rpm/tree/master/python/rpm
         #'rpm-python',
```

Comment:
Is this still needed? People trying to install rebase-helper on system Python will get an exception instructing them to install `python*-rpm`. How about replacing all this with something like:
```
This installs the rpm package if not already present
```

Reply:
Yeah, I agree with you. I will fix on your suggested way.